### ISSUE TITLE
How to force to use port 5671?

### ISSUE THREAD

**Issue by tom08zehn** [paolosalvatori/ServiceBusExplorer#811]
When I monitor the connection with Wireshark then I see that Service Bus Explorer always uses port 443. I tried various combinations for Connectivity Mode, Transport Type and AMQP Web Sockets.

Can I force Service Bus Explorer to use port 5671?

**Comment by SeanFeldman** [comment]
You're referring to the general Options?
![image](https://github.com/user-attachments/assets/972f8699-2ad0-477f-b07a-ad1c99f081f0)

IIRC, the old ASB client auto-detects what's available, and if TCP is blocked, it falls back to HTTP(s).

**Comment by tom08zehn** [comment]
Yes, I played with these as well but mean the (same) settings in the File -> Connect window.

**Comment by SeanFeldman** [comment]
Thanks to the 3 generations of the SDKs, there's a lot of confusion. My attempt to clarify that.

**Management operations** 
- Always via HTTP
- Latest SDK has an exception for Filters that can leverage the transport used for the messaging operations

Messating operational
- Latest SDK: single transport AMQP (default) or WebSockets (HTTPs), no auto-detection
- Previous SDKs - AMQP and HTTPs with auto-detection. This is what SBE is using

Protocol
- Always AMQP
- Version 1.0 (no support for 0.9)

Hope this helps.
